src/ipnet.rs

```diff
@@ -1073,6 +1189,119 @@ impl Iterator for Ipv6NetIter {
     }
 }
 
+/// Error conditions that can arise when building a ULA.
```

Comment:
```suggestion
/// Error conditions that can arise from [UlaBuilder::build]
```

Note we don't seem to having trailing periods on types and variants; we do for methods.